datatypes/src/main/java/org/hyperledger/besu/datatypes/Address.java

```diff
@@ -270,4 +270,10 @@ public boolean equals(final Object obj) {
     Address other = (Address) obj;
     return Arrays.equals(this.toArrayUnsafe(), other.toArrayUnsafe());
   }
+
+  @Override
+  public int hashCode() {
+    // Delegate to parent's hashCode implementation which uses delegate.hashCode()
+    return super.hashCode();
```

Comment:
```suggestion
     return Arrays.hashCode(this.toArrayUnsafe());
```

Reply:
ok have added the field. This has broken some tests in `PendingTransactionEstimatedMemorySizeTest` because it's added 40 to the size of the pending transaction (extra field on sender) - I guess this means that it would increase the txpool mem size? think we need to figure out if caching this value is worth making that change.